packages/server/src/utilities/rowProcessor/tests/utils.spec.ts

```diff
@@ -79,59 +60,4 @@ describe("rowProcessor utility", () => {
       expect(fixAutoColumnSubType(schema)).toEqual(schema)
     })
   })
-
-  describe("processAIColumns", () => {
```

Comment:
What was being tested here is covered by more realistic tests of AI columns in `row.spec.ts.